execution/executor-types/src/lib.rs

```diff
@@ -256,7 +258,7 @@ impl ExecutedTrees {
     pub fn version(&self) -> Option<Version> {
         let num_elements = self.txn_accumulator().num_leaves() as u64;
         if num_elements > 0 {
-            Some(num_elements - 1)
+            Some(num_elements.saturating_sub(1))
```

Comment:
Please do the `checked_x().expect()` thing here as well. Things can be copied around, where `num_elements > 0` can be no longer explicitly true.